pxr/base/tf/testenv/hash.cpp

```diff
@@ -294,6 +295,12 @@ Test_TfHash()
     std::vector<int> vint = {1, 2, 3, 4, 5};
     printf("hash(vector<int>): %zu\n", h(vint));
 
+    std::set<int> sint = {1, 2, 3, 4, 5};
+    printf("hash(set<int>): %zu\n", h(sint));
+
+    std::map<int, uint> mint = {{-1, 1}, {2, 3}, {-4, 5}};
```

Comment:
Not sure whether it will affect subsequent pr’s but noting we had to replace `uint` with `uint32_t` for windows, @nvmkuruc

Reply:
PR updated to use `uint32_t` and dependent PRs have all been updated.